vent/menu.py

```diff
@@ -114,7 +114,8 @@ class VentApp(npyscreen.NPSAppManaged):
 
     def onStart(self):
         """ Override onStart method for npyscreen """
-        self.addForm("MAIN", VentForm, name="Vent\t\t\t\t\t\t\t\tPress ^T to toggle help", color="IMPORTANT")
+        version = pkg_resources.require("vent")[0].version
```

Comment:
why not use the function that already gets the version?

https://github.com/CyberReboot/vent/blob/experimental/vent/helpers/meta.py#L5